packages/story-editor/src/components/publishModal/mainContent/storyPreview.js

```diff
@@ -16,20 +16,200 @@
 /**
  * External dependencies
  */
-import { Headline, THEME_CONSTANTS } from '@googleforcreators/design-system';
+import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import {
+  Button,
+  Headline,
+  Icons,
+  Text,
+  THEME_CONSTANTS,
+  BUTTON_VARIANTS,
+  BUTTON_TYPES,
+  BUTTON_SIZES,
+} from '@googleforcreators/design-system';
+import { PAGE_RATIO } from '@googleforcreators/units';
 import { __ } from '@googleforcreators/i18n';
+/**
+ * Internal dependencies
+ */
+import { useConfig, useStory } from '../../../app';
+
+// Set the available space for cover preview image + overlay
+const PreviewWrapper = styled.div`
+  width: ${({ width }) => width}px;
+  height: ${({ height }) => height}px;
+  position: relative;
+  margin-top: 8px;
+`;
+PreviewWrapper.propTypes = {
+  width: PropTypes.number.isRequired,
+  height: PropTypes.number.isRequired,
+};
 
 const PreviewContainer = styled.div`
-  margin: 8px 0;
-  object-fit: contain;
+  width: ${({ width }) => width}px;
+  height: ${({ height }) => height}px;
+  position: absolute;
+`;
+PreviewContainer.propTypes = {
+  width: PropTypes.number.isRequired,
+  height: PropTypes.number.isRequired,
+};
+
+// In ascending order, the featured image is set as the full available 2:3 space.
+export const Image = styled.img`
+  height: 100%;
+  width: 100%;
+  display: block;
+  aspect-ratio: auto ${PAGE_RATIO};
```

Comment:
ah I've never seen the `aspect-ratio: auto <ratio>` syntax. always thought it was either `auto` or `<ratio>`. what does the auto with the ratio there do?